src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/AuroraTopologyService.java

```diff
@@ -189,7 +197,9 @@ public void setClusterInstanceTemplate(HostInfo clusterInstanceTemplate) {
 
   private boolean refreshNeeded(ClusterTopologyInfo info) {
     Instant lastUpdateTime = info.lastUpdated;
-    return lastUpdateTime == null || Duration.between(lastUpdateTime, Instant.now()).toMillis() > refreshRateInMilliseconds;
+    return lastUpdateTime == null || Duration
```

Comment:
Maybe it conforms code style but it's unreadable for me.

Reply:
These line breaks are a side effect from using IntelliJ's reformat feature, not actually to conform to the checkstyle.